subsystems/shooter.py

```diff
@@ -0,0 +1,27 @@
+import wpilib
+import rev
+
+class shooter:
+    def __init__(self, port):
+        print("Shooter init")
+        self.stick = wpilib.XboxController(0)
+        self.motor = rev.CANSparkMax(port, rev.MotorType.kBrushed)
+        
+    def setShooterSpeed(self):
+      
+      forward = self.stick.getRawAxis(5)
```

Comment:
Do not check for input in the subsystem, do this in teliop and pass the value to the shooter class
Also, only X will be used for shooting, we will find the optimal speed after testing

Reply:
Use left trigger down + A button for manual shoot at 50% power.